README.md

```diff
@@ -1,9 +1,17 @@
 AeroGear Windows Cookbook
 =========================
 
-
 The AeroGear Windows cookbook is a list of recipes to quick start your Windows AeroGear experience.
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Visual Studio  |

```

Comment:
more a general question: is that the only "build tool" ? 

I know Apache NPanday :-) but that does not really make sense, IMO ...